BSM2 Flowsheet Documentation Update (#1354)
watertap-org/watertap: docs/technical_reference/flowsheets/extended_BSM2.rst

    Figure 1. BSM2 flowsheet

The objective function is to minimize the levelized cost of water, which can be represented by the following equation
where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor

> :math:`C_{ca,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and

Reviewer: ```suggestion
:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
```

Reviewer: My suggestion is to apply this to both rst files.

:math:`f_{util}` represents the utilization factor:

    .. math::

>         LCOW_{Q} = \frac{f_{crf}   C_{ca,tot} + C_{op,tot}}{f_{util} Q}

Reviewer: ```suggestion
        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
```

Reviewer: My suggestion is to apply this to both rst files.

    * `Modified ASM2d <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/modified_ASM2D.html>`_
    * `Modified ADM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/modified_ADM1.html>`_

Degrees of Freedom
------------------

> The following variables must be specified by the user to run the BSM2 flowsheet:

Reviewer: ```suggestion
The following variables must are initially specified for simulating the BSM2 flowsheet (i.e., degrees of freedom = 0):
```

Reviewer: My suggestion is to apply this to both rst files.

   "Dewatering unit hydraulic retention time", "1800", ":math:`\text{s}`"

   "**Thickener**"
   "Thickener hydraulic retention time", "86400", ":math:`\text{s}`"
   "Thickener diameter", "10", ":math:`\text{kWh/}\text{m}`"

Additional Variables
--------------------

.. csv-table::
   :header: "Description", "Symbol", "Value", "Units"

   "Reactor 5 oxygen mass transfer coefficient",":math:`KLa_{R5}`", "240", ":math:`\text{hr}^{-1}`"
   "Reactor 6 oxygen mass transfer coefficient",":math:`KLa_{R6}`", "240", ":math:`\text{hr}^{-1}`"
   "Reactor 7 oxygen mass transfer coefficient",":math:`KLa_{R7}`", "240", ":math:`\text{hr}^{-1}`"
   "Dissolved oxygen concentration at equilibrium",":math:`S_{O, eq}`", "8e-3", ":math:`\text{hr}^{-1}`"

Additional Constraints
----------------------

.. csv-table::
   :header: "Description", "Equation"

   "Reactor 5 mass transfer", ":math:`injection_{R5, S_{O2}} = KLa_{R5} * V_{R5} * (S_{O, eq} - S_{O, out})`"
   "Reactor 6 mass transfer", ":math:`injection_{R6, S_{O2}} = KLa_{R6} * V_{R6} * (S_{O, eq} - S_{O, out})`"
   "Reactor 7 mass transfer", ":math:`injection_{R7, S_{O2}} = KLa_{R7} * V_{R7} * (S_{O, eq} - S_{O, out})`"

> 

Reviewer: Still unsure whether we should include this Future refinements section. On the one hand, it let's the user know that the flowsheet isn't perfect and points out particular areas of improvement. On the other hand, we'll have to remember to update this section whenever relevant changes are applied, and I can imagine flowsheets where no future refinement is planned. The alternative would just be to track future refinements in an issue.

Let's see if @TimBartholomew or others have thoughts on this.
